### PR TITLE
Added Devra 2016 and Chapeau 2016 badges

### PR DESCRIPTION
**Pull request by Witaminer**

### CODE REVIEW

**Comment by DrOverdose** [comment]
shouldnt these be devra-2 and chapeau-2 to match similar formatting as moyer-2 and everything? just a thought.

**Comment by Witaminer** [comment]
Might be but don't have to be. For now, more popular names for new character badges are "Name Year". Leaving decision for master.

**Comment by sonyatvwebteam** [comment]
I thought of using badge year as well in pull request #53, but DrOverdose raised the very valid point of what happens if they introduce a 2nd NL-1331 badge in 2016? Then you have a problem. My vote is to use the current naming scheme as is, or if we're going to use the format "devra-2016", then make it "devra-2016-1" so that when Niantic does it (and I won't be surprised at all if they do, since consistency has never been their forte), the code is ready to handle it.

**Comment by therealklanni** [comment]
Let me know if this should be merged, or if I should wait.

**Comment by Witaminer** [comment]
Will change 2016 to 2 in two hours. Now I'm out.

**Comment by Witaminer** [comment]
Now it is ready with 1 and 2 numbering not with 2016. Ready to merge I suppose :-)

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "hubot-ingress",
   "description": "Ingress commands for Hubot",
-  "version": "0.19.0",
+  "version": "0.19.2",
```
Comment (therealklanni) [inline package.json]: Revert this change, please. Version number changes are handled by the package maintainer(s), not contributors. 👍

**Comment by therealklanni** [comment]
Your branch needs to be rebased on master before this can be merged. Also, please undo the version number change.

**Comment by EvoldicA** [comment]
Are you submitting a new pull from the current master?  I was looking forward to having this merged.
